PLGOPNS-402: Release of OpenCart 3.10.0 (#1272)
MultiSafepay/docs: content/integrations/ecommerce-integrations/opencart/changelog.md

aliases: [/integrations/opencart/changelog/]
---

## 3.10.0
Release date: May, 7th 2021

### Added
+ PLGOPNS-388: Introduce a new option in order statuses settings, which allows to ignore the notification according with the selected status

> + PLGOPNS-395: Add support for custom order description

Reviewer: ```suggestion
+ PLGOPNS-395: Add support for custom order descriptions

```

> + PLGOPNS-393: Add setting field, to turn direct payment gateways in redirect

Reviewer: ```suggestion
+ PLGOPNS-393: Add setting field, to turn direct payment gateways into redirect payment gateways

```

### Changed
+ PLGOPNS-389: Refactor to use simplified methods from the PHP-SDK

> + PLGOPNS-390: Refactor to replace all msp strings, variables names and methods names, and use multisafepay instead, across all the files within the plugin

Reviewer: ```suggestion
+ PLGOPNS-390: Refactor to replace all 'msp' string, variable, and method names, and use 'multisafepay' instead, across all the files within the plugin

```
